File: Example Code/01 Variables, Data Types, Input and Output/Math_SlopeCalculator.py
print( "SLOPE CALCULATOR" )

x1 = input( "FIRST coordinate pair, enter x: " )
y1 = input( "FIRST coordinate pair, enter y: " )

x2 = input( "SECOND coordinate pair, enter x: " )
y2 = input( "SECOND coordinate pair, enter y: " )

slopeNumerator = y2 - y1;
slopeDenominator = x2 - x1;

slope = slopeNumerator / slopeDenominator

print( "The slope is: " + str( slope ) + ", or " + str( slopeNumerator ) + "/" + str( slopeDenominator ) )

File: Example Code/01 Variables, Data Types, Input and Output/Math_SlopeCalculator.cpp
#include <iostream>
using namespace std;

int main()
{
    float x1, y1, x2, y2;
    cout << "SLOPE CALCULATOR" << endl;
    
    cout << "Enter first coordinate pair x, y: ";
    cin >> x1 >> y1;
    
    cout << "Enter second coordinate pair x, y: ";
    cin >> x2 >> y2;
    
    float slopeNumerator = y2 - y1;
    float slopeDenominator = x2 - x1;
    
    float slope = slopeNumerator / slopeDenominator;
    
    cout << "The slope is: " << slope
        << " or " << slopeNumerator << "/" << slopeDenominator
        << endl;
    
    return 0;
}
